src/library_webgpu.js

```diff
@@ -2052,7 +2052,7 @@ var LibraryWebGPU = {
     }
 
     var data = _memalign(16, mapped.byteLength);
-    HEAPU8.fill(0, data, mapped.byteLength);
+    zeroMemory(data, mapped.byteLength);
```

Comment:
This does not look NFC, as before we did
```js
HEAPU8.fill(0, data, mapped.byteLength);
```
while after we are effectively doing
```js
HEAPU8.fill(0, data, data + mapped.byteLength);
```
Perhaps it was wrong before?

Reply:
Oops, it was definitely wrong before.